### DISCUSSION TITLE
`t.inRange`

### DISCUSSION THREAD

**Discussion by Richienb** [avajs/ava#3315] [Ideas]
Many use cases:

https://github.com/sindresorhus/p-throttle/blob/0ed167a1ab735288d8f2e96eee520f5067d06fe0/test.js#L62-L65
https://github.com/sindresorhus/delay/blob/6779d5ac90a34ec692a07389c11649e66e8a92a7/test.js#L9
https://github.com/sindresorhus/p-settle/blob/d6fadc5fa26f3832c04791399dbb7d4c8349a9b5/test.js#L76
...

We can have a better user experience out-of-the-box by specifying a more helpful default failure message:

https://github.com/sindresorhus/unique-random/pull/24/files#diff-751968c81720bf8ac33ff2b74f4161024d0648243a245d868ff211c7e9accabdR4-R9

**Comment by novemberborn** [comment]
What would the interface be?

**Reply by Richienb** [reply]
Following [`in-range`](https://github.com/sindresorhus/in-range):
`t.inRange(number, {start, end})`

To whoever implements it: don't forget about BigInts!!
